packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -310,3 +293,10 @@ const bytesToHex = (buffer: ArrayBufferLike) => {
 		.map((b) => b.toString(16).padStart(2, "0"))
 		.join("");
 };
+
+const hashPath = async (path: string) => {
```

Comment:
+1

Reply:
leaving it for now because hashPath cannot be shared easily because one is using web crypto and the other is using node (and there's yet another hashPath in EWC in go lol)